src/components/field/Field.vue

```diff
@@ -33,6 +33,7 @@
             <b-field
                 :addons="false"
                 :type="newType"
+                :message="newMessage ? formattedMessage : ''"
```

Comment:
Are we able to cover this with an automated test?

Reply:
Yes. It can be done.

I can open another PR with tests.